src/views/ApartmentApp/index.js

```diff
@@ -2,8 +2,9 @@
 import React, { Component } from 'react';
 import 'date-fns';
 import { Grid, Card, CardHeader, CardContent, Button, Typography } from '@material-ui/core/';
-import GordonDialogBox from '../../components/GordonDialogBox';
+// import GordonDialogBox from '../../components/GordonDialogBox';
```

Comment:
As long as this unused import was removed in another branch, I won't worry about it here

Reply:
Ah! I see you removed it in #910. Great!